Update Main Page (README.md) Include Mission Statement and Vision (#40)
common-workflow-language/cwl-website: README.md

the elected [CWL leadership team](#Leadership_Team), however every-day project decisions are made by the
[CWL community](#Support) which is open for participation by anyone.

> ## Vision and Mission Statement
> 
> ### CWL Vision

Reviewer: ```suggestion
## Vision for the CWL Project
_What this community hopes to have accomplished in the future_
```

Author: Looks good.

Author: I merged these in.

> Researchers, scientists, and analysts share their batch data analysis workflows without technical barriers using an open standard.  Sharing workflows this way is a usual occurrence and seen as a typical way of working.  The workflows are complete and run on a variety of environments; and people re-use shared workflow descriptions and build new workflows from them. No vendor dominates the ecosystem

Reviewer: ```suggestion
Researchers, scientists, and analysts share their batch data analysis workflows without
technical barriers using an open standard.  Sharing workflows this way is a usual occurrence
and seen as a typical way of working.  The workflows are complete and run on a variety of
environments; and people re-use shared workflow descriptions and build new workflows from
them. No vendor dominates the ecosystem.
```

Line wrapping, added a trailing period to the last sentence.

Author: Looks good.

> ### CWL Mission Statement 
> 
> To support open consensus-based standards for command line data analysis workflows and tools.
> 
> Specifically, we support the
> * pre-standards process by providing a neutral place of convening to discuss, propose and test ideas about workflow standards
> * standardization process by stewarding the development and delivery of standards in accordance with the [Open Stand principles](https://open-stand.org/about-us/principles/).  
> * post-standards life cycle by promoting the released standard, developing and maintaining related training and tools, and by tracking deficits and other post-standardization feedback.

Reviewer: ```suggestion
### Mission of the CWL Project
_How we plan to achieve our vision_

The CWL project supports open consensus-based standards for command line data analysis
workflows and tools.

Specifically, we support the
* pre-standards process by providing a neutral place of convening to discuss, propose and
test ideas about command-line tool based workflow standards and related topics
* standardization process by stewarding the development and delivery of standards in
accordance with the [Open Stand principles](https://open-stand.org/about-us/principles/).  
* post-standards life cycle by (1) promoting the released standards, (2) developing and maintaining
related training and tools, and by (3) tracking deficits and other post-standardization feedback.
```

Line wrapping, added "command-line tool based" before "workflows" to narrow the scope. Added "and related topics" to expand the scope a bit as well :-)

I think I will also move the OpenStand logo down to this section
"release standards" to "released standard**s**"

Author: Looks good.
